src/libsync/discovery.cpp

```diff
@@ -1400,9 +1396,25 @@ void ProcessDirectoryJob::processFileAnalyzeLocalInfo(
 
         return true;
     };
+    // BUGFIX: DATALOSS!!!
+    // TODO: Server implementation for E2EE files MOVE is REQUIRED!!! Currently, the E2EE MOVE is performed via DELETE and PUT, which is not compatible with
+    // online only files
```

Comment:
please
are you sure those comments are really useful ?

Reply:
to be clear, I would expect automated tests to explain that, not comments in capital letters